.changeset/tall-baths-dance.md

```diff
@@ -0,0 +1,28 @@
+---
+"@comet/admin": minor
+---
+
+**Toolbar:** add `topBarActions` prop to render custom actions in the top bar, and introduce `<HelpDialogButton />` for a built-in help dialog trigger.
+
+### What’s new
+
+- `Toolbar` now supports a new prop: `topBarActions` (class key & slot name: `"topBarActions"`). Use it to place action controls on the right side of the top bar.
+- New component: `HelpDialogButton` (exported from `@comet/admin`). It renders an icon button that toggles a modal dialog with a title and rich help content.
```

Comment:
Since this is a changelog explicitly for `@comet/admin`, I think that info can be omitted


```suggestion
- New component: `HelpDialogButton`. It renders an icon button that toggles a modal dialog with a title and rich help content.
```